api/src/main/resources/workbench-api.yaml

```diff
@@ -235,6 +235,35 @@ paths:
       responses:
         204:
           description: Request Received.
+
+  "/v1/getTerraTermsOfServiceStatus":
```

Comment:
we should probably remove "terra" here, because this endpoint should check *both* AoU and Terra TOS statuses.